src/traffic_server/CMakeLists.txt

```diff
@@ -15,6 +15,16 @@
 #
 #######################
 
+add_library(inkcont InkContInternal.cc)
```

Comment:
I think we should be explicit about STATIC here to avoid surprises if anyone sets BUILD_SHARED_LIBS